src/js/page_sections/Menu/Menu.js

```diff
@@ -55,8 +84,8 @@ class Menu extends Component {
           </button>
           <ul className="coa-Menu__list">
             <HomeMobileListItem handleClick={this.props.toggleMenu} />
-            <AirportMobileListItem />
-            <ThreeOneOneMobileListItem />
+            <AirportMobileListItem {...this.props} />
```

Comment:
can we pass just the intl provider instead of all props so it's more obvious what we actually need/use in the sub components.